day_6/exercises/burrito.js

```diff
@@ -16,12 +16,25 @@ class Burrito {
     this.base = base;
     this.toppings = toppings;
   }
+
+  addTopping() {
+    console.log(this.toppings + " peppers")
+  }
+
+  removeTopping() {
```

Comment:
same thing here. I'd like to see a topping be removed from the array in this method